cumulus/bridges/primitives/header-chain/tests/tests.rs

```diff
@@ -1,3 +1,19 @@
+// Copyright 2020-2021 Parity Technologies (UK) Ltd.
```

Comment:
```suggestion
// Copyright Parity Technologies (UK) Ltd.
```

There should be no dates anymore.

Reply:
Ok, I removed all the dates I could find.